llvm/lib/SYCLLowerIR/CompileTimePropertiesPass.cpp

```diff
@@ -619,6 +619,12 @@ bool CompileTimePropertiesPass::transformSYCLPropertiesAnnotation(
     if (*Property.first == "sycl-alignment")
       continue;
 
+    // leave these annotations as is. They will be processed by SPIRVWriter.
+    if (*Property.first == "sycl-prefetch-hint" ||
```

Comment:
I assume the type of `Property` is an `std::pair`. If so, please change line 616 to use structured bindings. Can be done either here or in another PR merged before this one.

Reply:
Is it what you assume to see [30e531a](https://github.com/intel/llvm/pull/11458/commits/30e531a17d6d59e337a85a36c22276be7d81ca10)? Or should we give some another names to _first_ and _second_ variables.

Reply:
Almost... Now need to change "first" and "second" into something meaningful.